test/parser/newline.js

```diff
@@ -51,10 +51,10 @@ describe('The parser', function () {
       var data = ['abcd', 'efgh']
 
       helper.expectData(lineParser, data, done)
-
-      var wholeBuffer = new Buffer(8 * 3 + 3 + testData.verseOne.length + testData.verseTwo.length + testData.rest.length)
-      helper.buildBuffer('abcd\n').copy(wholeBuffer)
-      helper.buildBuffer('efgh\n').copy(wholeBuffer, 8 + 5)
+      //                           header  content
+      var wholeBuffer = new Buffer(8 * 2 + 2 * 5)
+      helper.buildBuffer(data[0] + '\n').copy(wholeBuffer)
+      helper.buildBuffer(data[1] + '\n').copy(wholeBuffer, 8 + 5)
       helper.writeChunks(lineParser, [wholeBuffer])
     })
```

Comment:
I am ok with the change,
the line:

var wholeBuffer = new Buffer(8 * 3 + 3 + testData.verseOne.length +
testData.verseTwo.length + testData.rest.leng

should not have been there in the first place :)

On Sun, Feb 7, 2016 at 9:04 PM, Matteo Collina <notifications@github.com>
wrote:

> In test/parser/newline.js
> <https://github.com/mcollina/docker-loghose/pull/10#discussion_r52124794>:
>
> > @@ -51,10 +51,10 @@ describe('The parser', function () {
> >        var data = ['abcd', 'efgh']
> >
> >        helper.expectData(lineParser, data, done)
> > -
> > -      var wholeBuffer = new Buffer(8 * 3 + 3 + testData.verseOne.length + testData.verseTwo.length + testData.rest.length)
> > -      helper.buildBuffer('abcd\n').copy(wholeBuffer)
> > -      helper.buildBuffer('efgh\n').copy(wholeBuffer, 8 + 5)
> > +      //                           header  content
> > +      var wholeBuffer = new Buffer(8 * 2 + 2 * 5)
> > +      helper.buildBuffer(data[0] + '\n').copy(wholeBuffer)
> > +      helper.buildBuffer(data[1] + '\n').copy(wholeBuffer, 8 + 5)
> >        helper.writeChunks(lineParser, [wholeBuffer])
> >      })
>
> @cammellos <https://github.com/cammellos> what do you think? You wrote
> this test in the first place :).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mcollina/docker-loghose/pull/10/files#r52124794>.
>